logic/CMakeLists.txt

```diff
@@ -52,6 +61,20 @@ SET(LOGIC_SOURCES
 	wonko/OpSys.h
 	wonko/OpSys.cpp
 
+	# Resources
```

Comment:
I still feel this whole thing is like shooting mosquitos with an orbital death laser.